hardware/setup/create_framework.tcl

```diff
@@ -304,6 +309,11 @@ if { $fpga_card == "ADKU3" } {
     add_files -fileset constrs_1 -norecurse $root_dir/setup/AD8K5/snap_ddr4_b0pins.xdc
     set_property used_in_synthesis false [get_files $root_dir/setup/AD8K5/snap_ddr4_b0pins.xdc]
   }
+} elseif { ($fpga_card == "RCXVUP") } {
```

Comment:
Could be combined with the statement below...

Reply:
combined